README.md

```diff
@@ -1,83 +1,11 @@
-Takeaway Challenge
-==================
-```
-                            _________
-              r==           |       |
-           _  //            |  M.A. |   ))))
-          |_)//(''''':      |       |
-            //  \_____:_____.-------D     )))))
-           //   | ===  |   /        \
-       .:'//.   \ \=|   \ /  .:'':./    )))))
-      :' // ':   \ \ ''..'--:'-.. ':
-      '. '' .'    \:.....:--'.-'' .'
-       ':..:'                ':..:'
+My code accomplishes all the user stories and sends a text via Twilio on confirmation. 
 
- ```
+Process -->
 
-Instructions
--------
+I knew that I wanted the order class to be dependent on the menu and the menu on order and order on dish, so I wrote each class in reverse order beginning with the dish. I injected dependency as I went and used mocking to isolate tests in rspec (or to the best that I could)
```

Comment:
This is great idea to work in reverse order and helps keep things clear & simple.